src/api-reference/financial-integration/v4.financial-integration.markdown

```diff
@@ -401,6 +462,123 @@ Content-Length: 405
   }
 ]
 ```
+## <a name="payment-confirmations"></a>Post Financial Payment Confirmations
+
+This API allows financial payment results to be sent to Concur.
+
+### Scopes
+
+`FISVC` - Refer to [Scope Usage](#scope-usage) for full details.
+
+### Request
+
+#### URI
+
+##### Template
+
+```shell
+https://{datacenterURI}/financialintegration/fi/v4/companies/transactiontypes/{docType}/transactions/paymentconfirmations
+```
+
+##### Parameters
+
+Name|Type|Format|Description
+---|---|---|---
+`docType`|`string`|`path`|**Required** The financial document type to return. VALUES: expense; invoice; cashadvance; payroll; obligation. Only one type of transactions can be retrieved at a time
+`confirmationRequest`||[body](#confirmationrequest)|**Required** The JSON request to be posted
+
+#### Headers
+
+* [RFC 7235 Authorization](https://tools.ietf.org/html/rfc7235#section-4.2) - Bearer Token that identifies the caller. This is the Company access token.
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+
+#### Payload
+
+[Confirmation Request](schema-paymentrequest)
```

Comment:
Swagger doesn't show any content in this request, only a response. I suspect that the markdown is correct, but need your confirmation.

Reply:
true, swagger doesn't show content in request for posting APIs.

Reply:
I've added a comment in the Swagger PR asking for it to be updated with this request info.